static-web/src/main/resources/script.js

```diff
@@ -369,10 +378,30 @@ function connectWebsocket(gamekey, playerID) {
             $(".playerPlayground").css({display: "block"});
             $("body").css({background: "#ffffff"})
             $(".infoDialog").css({display: "none"});
+            $(".averageRating").css({display: "none"});
+
             showPlayersNewTablemaster(message);
             getAllPlayers(gamekey, playerID);
         }
         if (data.data.startsWith("IAmNowTheOneAndOnlyTablemaster")) {
+            var messageSplit = data.data.split(",")
+            var isGameRunning = messageSplit[1];
+
+            if (isGameRunning === "true") {
+                $.ajax({
```

Comment:
doing an AJAX call here could lead to the container ".averageRating" being displayed at the start of the game imo, because it will trigger a websocket message of the backend with the following callback of the frontend:
```js
$('.cardsFrontSide').css({display: "none"});
$('.cardsBackSide').css({display: "flex"});
getAllPlayers(gamekey, playerID);
showAverage(gamekey); // <- 
```
when the game is over, there are no other changes to the frontend than the above

the backend needs some configuration when the game is over, see method
```java 
public void gameOver(String gamekey) throws Exception
```
only `setRunning(gamekey, false);` inside here is relevant for the backend, so one could do this call inside
```java
 public void setUpNewTablemaster(String gamekey, Long playerID, WebSocketSession session) throws Exception
```
for example